deploy/osd-delete-backplane-serviceaccounts/10-delete-backplane-serviceaccounts.rbac.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: openshift-backplane
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-delete-serviceaccounts
```

Comment:
```suggestion
  name: osd-delete-backplane-serviceaccounts
```

Reply:
Have update name here and in subjectpermisison